main.go

```diff
@@ -196,6 +210,12 @@ func wrappedMain() int {
 	}
 
 	exitCode, err := cli.Run()
+	if !inPlugin {
+		if err := packer.CheckpointReporter.Finalize(cli.Subcommand(), exitCode, err); err != nil {
+			log.Printf("Error finalizing telemetry report. This is safe to ignore. %s", err.Error())
```

Comment:
Should we pre-emptively make this message adhere to ENG-003 so that when we make a switch to the right logging lib we have the right prefixes?